docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+           "english": {}
+       }
+   }
+   ```
+
+
+### 4. Add your decimal and text
+- Inside the language brackets, paste your <Tooltip text={<strong>decimal number</strong>} tooltip="A string converted into a numeral form" /> in "quotes"
+- Add your new text... Also in "quotes".
+
+Result:
+```json
+"localisationOverrides": {
+    "yourLOCRhash": {
+        "english": {
+            "yourconvertednamedecimal": "New Suit Name"
+        }
+    }
+}
+````
+<div id="syntax">
+<details>
+<summary>**Syntax**</summary>
+:::info Syntax
+If you're paying attention, you might notice that we're setting up the localisation override code in the manifest just like our <Tooltip text={<strong>LOCR File</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." /> 
+where we got the <Tooltip text={<strong>strings</strong>} tooltip= "A key that links to text in the LOCR File" /> from. 
+Both will be using the same <Tooltip text={<strong>'Syntax'</strong>} tooltip="A set of rules that define how code is written and organized." />.
+
+> This was visually represented as a *simplified LOCR file* at the [beginning of Part 2](#pt-2-get-your-string).
+
+
+*Even better too if you have noticed how similar this method is to [the previous *blobs article*](addingblobs.md#step-3-set-up-a-blobsfolder) where we 'recreated'* <Tooltip text={<strong>an assembly path</strong>} tooltip="Bonus points if you connected that HASH lines are 'coded' assembly paths = filePaths + fileNames." />
+:::
+</details>
+</div>
+
+### 5. (Optionally) Add more languages
+#### Same Strings:
+In [Part 2](#2-copy-the-string) of the tutorial we demonstrated that *strings* are the same in each <Tooltip text={<><strong>language array</strong><span className="icon-alert"></span></>} tooltip="A single language array in a LOCR file consisting of strings" />
+>*This will make adding languages for each entry very easy.*
+    
+
+To have your mod support more languages, simply:
+- make another <Tooltip text={<strong>language bracket</strong>} tooltip="A single language array that can contain multiple entries of strings each attached to text you see in-game" /> 
+- add the <Tooltip text={<strong>same decimal</strong>} tooltip="A string converted into a numeral form" /> with your text
+
+
+Example of a new language line:
+<div id="langcode"></div>
+```json
+"localisationOverrides": {
+        "yourLocrHASH": {
+            "english": {
+                "yourconverteddecimal": "New Suit Name"
+            },
+            "french": {
+                "yourconverteddecimal": "Niveua Costume nome"
+            }
+        }
+    }
+```
+<details>
+<summary>Language Codes</summary>
+
+:::info language codes
+- The reason why `french` is used here and not `français` is because the manifest's <Tooltip text={<strong>schema</strong>} tooltip="A structure used to validate and provide auto-completion for your manifest, helping you write error-free code." /> is coded in English. 
```

Comment:
I feel like they would have noticed the manifest is written in English a long term before this point